index.bs

```diff
@@ -296,7 +298,7 @@ The API is defined by the following Web IDL fragment.
 
     interface ScopedCredentialInfo {
         readonly attribute Credential           credential;
-        readonly attribute any                  publicKey;
+        readonly attribute CryptoKey            publicKey;
```

Comment:
This seems bad from an API usability perspective though. It makes the IDL more elegant but usage harder.
- Return JsonWebKey (represented in WebIDL as any for technical reasons): Caller can simply call JSON.serialize on publicKey and send it to server.
- Return CryptoKey: Caller gets CryptoKey, then must know to call WebCrypto's exportKey method to export the key for sending to the server.
Considering that the most common (perhaps only) usage of the publicKey here is to upload to the server for registration, why would we force the programmer to learn about WebCrypto for this one thing?